web/src/main/react-dashboard/src/constants.js

```diff
@@ -3,3 +3,4 @@ export const ERROR = "ERROR";
 export const SUCCESS = "SUCCESS";
 export const RUNNING = "RUNNING";
 export const PENDING = "PENDING";
+export const TESTGRID_CONTEXT = "/testgrid/v0.9";
```

Comment:
This is the old context. New context is /testgrid/dashboard/. We already have a constant for this I think. 
cc @sameerawickramasekara

Reply:
I think you are referring to [1]. I've checked with that context. But some parts of the UI was not loaded properly. Because of that, I've decided to go with this context. Will check it again.

[1]. https://github.com/wso2/testgrid/blob/master/web/src/main/react-dashboard/src/components/ProductStatusView.js#L39

Reply:
Approved the changes.